testhelper/docker/resource/postgres/postgres.go

```diff
@@ -27,6 +29,9 @@ type Resource struct {
 	User     string
 	Host     string
 	Port     string
+
+	ContainerName string
+	ContainerID   string
```

Comment:
How are you planning to use these in your tests?

Reply:
This is information current postgres test already uses to stop the container. Could be used for logging as well